### PR TITLE
Extracted code to read file into a reusable function

### PR DESCRIPTION
**Pull request by FedeIII**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -64,6 +64,8 @@ bld/
 .vs/
 # Uncomment if you have tasks that create the project's static files in wwwroot
 #wwwroot/
+#VS Code
+.vscode
```
Comment (FedeIII) [inline .gitignore:68]: Ignore vscode config folder

File: Aster/Public/Physics.h [hunk 2]
```diff
@@ -29,7 +29,7 @@ class Physics
 		glm::vec3 size = glm::vec3(0.0f);
 
 		float mass = 0.0f;
-		glm::vec3 accelerationForce = glm::vec3(0.0f);;
+		glm::vec3 accelerationForce = glm::vec3(0.0f);
```
Comment (FedeIII) [inline Aster/Public/Physics.h:32]: Fixed extra `;`

File: Aster/Textures/player_walk.txt [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
-1921,1,128,128
-2049,1,128,128
-2177,1,128,128
-2305,1,128,128
-2433,1,128,128
-2561,1,128,128
-2689,1,128,128
-2817,1,128,128
\ No newline at end of file
+1 1 128 128
```
Comment (FedeIII) [inline Aster/Textures/player_walk.txt:1]: Following same standard as one.lvl so the read files function is easier to implement

File: Files.cpp [hunk 4]
```diff
@@ -0,0 +1 @@
+#include "files.h"
```
Comment (thenanox) [inline Files.cpp]: what is this file doing at this level?

**Review by thenanox — APPROVED** [review]

**Pull request merged**